README.md

```diff
@@ -1,6 +1,15 @@
-# Naptime Skill
+## Nap time
+A skill to put the Mycroft listener to sleep.
+
+## Description 
+While asleep Mycroft will not react to any queries (and will not send any sentences for parsing by the STT) until the device is awoken by the words "Hey Mycroft, Wake up".
+
+## Examples 
+* "go to sleep"
```

Comment:
For the examples, let's just use:
> * "Go to sleep"
> * "Nap time"
> * "Wake up"

The "sleepytime" thing is odd -- the actual words are 'sleepytime tea'.  I'm guessing this was an inside joke reference to the tea, but is fairly unnatural to say.  We can leave it as an easter egg, I guess.